### PR TITLE
SEO-183463 MAUI Missing H1

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,
Have worked on Missing H1, please review.

Regards,
Yvone.

### CODE REVIEW

File: MAUI/CheckBox/Overview.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: Learn here about overall key features in Essential Studio for .NET
 control: SfChekBox
 ---
 
-## .NET MAUI CheckBox (SfCheckBox) Overview
+# .NET MAUI CheckBox (SfCheckBox) Overview
```
Comment (gowthamanthavasiyappan) [inline MAUI/CheckBox/Overview.md:9]: Change like this - Overview of .NET MAUI CheckBox (SfCheckBox)

**Review by gowthamanthavasiyappan — COMMENTED** [review]
Commented

File: MAUI/CheckBox/Overview.md (hunk 1 above)
Comment (Yvone-Atieno) [reply MAUI/CheckBox/Overview.md]: changed

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**